charts/s3gw/templates/ui-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.ui.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: '{{ .Chart.Name }}-ui'
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.aquarist-labs.io/name: '{{ .Chart.Name }}-ui'
+    helm.sh/chart: '{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}'
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.aquarist-labs.io/name: '{{ .Chart.Name }}-ui'
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app.aquarist-labs.io/name: '{{ .Chart.Name }}-ui'
+    spec:
+      containers:
+        - name: s3gw-ui
+          image:
+            '{{ .Values.imageRegistry }}/{{ .Values.imageName_ui }}:{{ .Values.imageTag_ui }}'
+          imagePullPolicy: Never
```

Comment:
We should make this parametric; if this is pulled from a registry it needs to be different from `Never`

Reply:
Thanks, this was an oversight. I should have been `{{ .Values.imagePullPolicy }}`, which is also the pull policy for the s3gw image.